src/main/java/hudson/plugins/sshslaves/verifiers/ManuallyProvidedKeyVerificationStrategy.java

```diff
@@ -92,8 +94,16 @@ private static HostKey parseKey(String key) {
                 RSASHA1Verify.decodeSSHRSAPublicKey(keyValue);
             } else if ("ssh-dss".equals(algorithm)) {
                 DSASHA1Verify.decodeSSHDSAPublicKey(keyValue);
+            } else if ("ssh-ed25519".equals(algorithm)) {
```

Comment:
Sadly, this doesn't seem to work. Getting this build failure (see latest update to PR):

```
[ERROR] /scratch/jenkins/workspace/plugins/ssh-slaves-plugin/src/main/java/hudson/plugins/sshslaves/verifiers/ManuallyProvidedKeyVerificationStrategy.java:[96,43] cannot find symbol
[ERROR] symbol:   class ED25519KeyAlgorithm
[ERROR] location: package com.trilead.ssh2.signature
[ERROR] /scratch/jenkins/workspace/plugins/ssh-slaves-plugin/src/main/java/hudson/plugins/sshslaves/verifiers/ManuallyProvidedKeyVerificationStrategy.java:[98,43] cannot find symbol
[ERROR] symbol:   class ECDSAKeyAlgorithm
[ERROR] location: package com.trilead.ssh2.signature
[ERROR] /scratch/jenkins/workspace/plugins/ssh-slaves-plugin/src/main/java/hudson/plugins/sshslaves/verifiers/ManuallyProvidedKeyVerificationStrategy.java:[100,43] cannot find symbol
[ERROR] symbol:   class ECDSAKeyAlgorithm
[ERROR] location: package com.trilead.ssh2.signature
[ERROR] /scratch/jenkins/workspace/plugins/ssh-slaves-plugin/src/main/java/hudson/plugins/sshslaves/verifiers/ManuallyProvidedKeyVerificationStrategy.java:[102,43] cannot find symbol
[ERROR] symbol:   class ECDSAKeyAlgorithm
[ERROR] location: package com.trilead.ssh2.signature
```

Thoughts? I haven't written Java in ~10 years, so forgive my n00bism here. :)